be/src/util/url_coding.cpp

```diff
@@ -122,59 +122,47 @@ static void encode_base64_internal(const std::string& in, std::string* out,
     out->assign((char*)buf.get(), d - buf.get());
 }
 
-void base64url_encode(const std::string& in, std::string *out) {
+void base64url_encode(const std::string& in, std::string* out) {
     static unsigned char basis64[] =
             "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789-_";
     encode_base64_internal(in, out, basis64, false);
 }
 
 void base64_encode(const std::string& in, std::string* out) {
     static unsigned char basis64[] =
-        "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789+/";
+            "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789+/";
     encode_base64_internal(in, out, basis64, true);
 }
 
-static char encoding_table[] = {
-    'A', 'B', 'C', 'D', 'E', 'F', 'G', 'H',
-    'I', 'J', 'K', 'L', 'M', 'N', 'O', 'P',
-    'Q', 'R', 'S', 'T', 'U', 'V', 'W', 'X',
-    'Y', 'Z', 'a', 'b', 'c', 'd', 'e', 'f',
-    'g', 'h', 'i', 'j', 'k', 'l', 'm', 'n',
-    'o', 'p', 'q', 'r', 's', 't', 'u', 'v',
-    'w', 'x', 'y', 'z', '0', '1', '2', '3',
-    '4', '5', '6', '7', '8', '9', '+', '/'
-};
+static char encoding_table[] = {'A', 'B', 'C', 'D', 'E', 'F', 'G', 'H', 'I', 'J', 'K', 'L', 'M',
```

Comment:
IMO, format files which are copied from other projects is not a good idea, because when we want to sync code from source project which have fixed some bugs or add some new features, there will be much diff and conflicts to resolve.

Reply:
If there is some diff, maybe it is the resposibility for author who copied the file to fix it, Even if the code is formated, we can found the code that should modified.I think unify the codestyle in one project maybe more important.